website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Concepts"
+linkTitle: "Concepts"
+weight: 35
+---
+
+By adding Karpenter to a Kubernetes cluster you can dramatically improve the efficiency and cost of running workloads on that cluster.
+Instead of introducing lots of new concepts and configuration, Karpenter is tightly integrated with Kubernetes features to make sure that the right types and amounts of compute resources are available to pods as they are needed.
+Karpenter carries out its duties by:
+
+* Watching for pods that the Kubernetes scheduler has marked as unschedulable
+* Evaluating the requirements (capacity, features, and taints) identified by the unscheduable pods
+* Provisioning nodes that meet the requirements of the unschedulable pods
+* Scheduling the unschedulable pods to run on the new nodes
+* Removing the nodes when the nodes are no longer needed
+
+In many cases, you can configure a simple Karpenter provisioner when it is first installed and not change it again. 
+Other times, you might want to continue to tweak the provisioner or even create multiple provisioners for a cluster where you tailor the requirements of different teams to use specific instance types, zones, or computer architectures.
+Likewise, you might want to instruct the provisioner to take advantage of underlying cloud features, such as AWS spot instances.
+
+As someone deploying pod workloads, you can make specific requests for capacity and features you want from the nodes running your pods.
```

Comment:
> As someone deploying pod workloads

I'd love to standardize on terms. Perhaps cluster operator, application developer, end user?

I also find the framing a bit odd here. You're not requesting capacity for its own sake, you're trying to run pods. 

I might say something like, "node features and capacity are determined dynamically based on the requirements of your pods".

Reply:
We might be able to simply omit end users, since they shouldn't interact much with our layer of abstraction.